include/swift/AST/DiagnosticsSema.def

```diff
@@ -327,13 +327,13 @@ ERROR(cannot_throw_error_code,none,
       "instance", (Type, Type))
 
 ERROR(cannot_throw_nil,none,
-      "cannot infer concrete Error for thrown 'nil' value", ())
+      "cannot infer concrete Error for thrown 'nil value", ())
 
```

Comment:
Did you mean to remove only the closing `'` here?

Reply:
Thanks.